iot_input_oca/views/iot_device_input_views.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="iot_device_input_tree" model="ir.ui.view">
+        <field name="name">iot.device.input.tree</field>
+        <field name="model">iot.device.input</field>
+        <field name="arch" type="xml">
+            <tree delete="0">
+                <field name="name" />
+                <field name="address" />
+                <field name="serial" />
+            </tree>
+        </field>
+    </record>
+    <record id="iot_device_input_form" model="ir.ui.view">
+        <field name="name">iot.device.input.form</field>
+        <field name="model">iot.device.input</field>
+        <field name="arch" type="xml">
+            <form>
+                <header />
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
```

Comment:
The new logic is made using the web_ribbon

Reply:
Thank you! :+1: